extension/src/cli/dvc/runner.ts

```diff
@@ -208,69 +210,4 @@ export class DvcRunner extends Disposable implements ICli {
       cwd
     })
   }
-
-  private notifyOutput(process: Process) {
-    process.all?.on('data', chunk =>
-      this.processOutput.fire(
-        (chunk as Buffer)
-          .toString()
-          .split(/(\r?\n)/g)
-          .join('\r')
-      )
-    )
-  }
-
-  private notifyCompleted({
-    command,
-    pid,
-    cwd,
-    duration,
-    exitCode,
-    killed,
-    stderr
-  }: CliResult & {
-    killed: boolean
-  }) {
-    this.processCompleted.fire({
-      command,
-      cwd,
-      duration,
-      exitCode,
-      pid,
-      stderr: stderr?.replace(/\n+/g, '\n')
-    })
-
-    this.sendTelemetryEvent({ command, duration, exitCode, killed, stderr })
-  }
-
-  private sendTelemetryEvent({
-    command,
-    exitCode,
-    stderr,
-    duration,
-    killed
-  }: {
-    command: string
-    exitCode: number | null
-    stderr?: string
-    duration: number
-    killed: boolean
-  }) {
-    const properties = { command, exitCode }
-
-    if (!killed && exitCode && stderr) {
-      return sendErrorTelemetryEvent(
-        EventName.EXPERIMENTS_RUNNER_COMPLETED,
```

Comment:
[F] This event has been removed. We don't do this for any other processes that we start.